Cargo.toml

```diff
@@ -37,10 +38,17 @@ url = "2.5"
 holochain_http_gateway = { path = ".", features = ["test-utils"] }
 
 assert2 = "0.3.15"
-holochain = { version = "0.4.1", features = ["sweettest"] }
+holochain = { version = "0.4.1", default-features = false, features = [
+  "sqlite-encrypted",
+  "tx5",
+  "wasmer_sys",
+  "sweettest",
+] }
 http-body-util = "0.1.2"
 mockall = "0.13"
-reqwest = { version = "0.12", features = ["rustls-tls"] }
+reqwest = { version = "0.12", default-features = false, features = [
+  "rustls-tls",
```

Comment:
It looks like we're not using TLS anywhere in the tests. Can be dropped too?

Reply:
Oh yeah, du'h :) Fixed